NSS-Day-Cohort-53/tabloid-full-stack-always-picturesque-icebergs
# Comments can be viewed from Post Details

Referencing ticket #29 

#### Changes Made
1. Added file `Comments/Comments.js` to the `components` directory.
2. Modified file `authManager.js` to include get comments fetch call.
3. Modified file `ApplicationViews.js` to include get comments page routing.
4.  Modified file `PostDetails.js` to include get "View Comments" button.
5.  Added the `Comment.cs` class to `Models` directory.
6.  Modified the `Post.js` class to contain the Comments property.
7.  Modified files `PostRepository.cs` and `IPostRepository.cs` to contain query methods.
8. Modified file `PostController.cs` to handle Controllers HTTP requests.


#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout ma-ViewComments
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
4. Test app functionality.
    As a logged-in user:
    >Click Posts in the navbar, click on the title of a particular post

    >Click the "View Comments" button - you should see comments listed (recent first) followed by username and create date
    
    >Click "Return to Post" and you should be taken back to post details
5. View code file.
    > Confirm file modifications are present as indicated above.
    
   > Confirm no unused code or extraneous comments exist.
